src/marten/db/field/many_to_many.cr

```diff
@@ -120,8 +122,8 @@ module Marten
               register_field(
                 {{ @type }}.new(
                   {{ field_id.stringify }},
-                  {% unless kwargs.is_a?(NilLiteral) %}**{{ kwargs }}{% end %},
```

Comment:
This line is not related to the main topic, but I think comma should be present inside the  block. It worked because kwargs is not nil, because `to` is required.